elements/pfe-card/README.md

```diff
@@ -54,6 +54,7 @@ There are several attributes available for customizing the visual treatment of t
 - `pfe-img-src`: Optional background image applied to the entire card container.  Alignment of this image can be managed using the `--pfe-card--BackgroundPosition` variable which is set to `center center` by default.
 - `pfe-size`: Optionally adjusts the padding on the container.  Accepts: `small`.
 - `pfe-overflow`: Optionally allows an image or element to overflow the padding on the container. This property should be added to the direct child of the slotm such as on an image tag; should be added to the element that you want to overflow the container. Accepts: `top`, `right`, `bottom`, `left`.
+- `pfe-border`: Optionally apply a border color and weight to the entire card container. The default color and weight is `#d2d2d2` and `1px`, respectively.
```

Comment:
Could we also add notes about the other new vars:

```md
- **Spacing** You can adjust the spacing between the regions...
- **Card header background color** You may set the background color of the card independent of the card itself with this variable:  `--pfe-card__header--BackgroundColor`
```

Reply:
Those are custom properties and not attributes (this section is for attributes only).  There is a section below that outlines custom properties in a general sense though I do like the idea of having a whole table of available properties.

Reply:
@starryeyez024 Added a new properties table below.